datatypes/src/collections/feature_collection.rs

```diff
@@ -1161,7 +1137,7 @@ mod tests {
         for i in 0..10 {
             assert_eq!(
                 gen_collection(i).byte_size(),
-                empty_hash_map_size + struct_stack_size + 264 + time_interval_size(i)
+                empty_hash_map_size + struct_stack_size + 192 + time_interval_size(i)
```

Comment:
why did this magic number change?

Reply:
It seems like the calculation method changed, I copy-pasted it from their master branch.

Reply:
76b4180